extensions/amp-dailymotion/0.1/amp-dailymotion.js

```diff
@@ -96,10 +271,72 @@ class AmpDailymotion extends AMP.BaseElement {
 
   /** @override */
   pauseCallback() {
-    if (this.iframe_ && this.iframe_.contentWindow) {
-      this.iframe_.contentWindow./*OK*/postMessage('pause', '*');
+    if (this.iframe_ && this.iframe_.contentWindow
+       && this.playerState_ == DailymotionEvents.PLAY) {
+      this.pause();
     }
   }
+
+  /**
+   * @override
+   */
+  play(isAutoplay) {
+    this.sendCommand_('play');
+    // Hack to solve autoplay problem on Chrome Android
+    // (first play always fails)
+    if (isAutoplay && this.playerState_ != DailymotionEvents.PAUSE) {
+      this.loadedPromise_.then(() => {
```

Comment:
I renamed the event from `DailymotionEvents. LOADED` to `DailymotionEvents. STARTED_BUFFERING` in the last commit